sympy/printing/octave.py

```diff
@@ -432,6 +435,22 @@ def _print_LambertW(self, expr):
         return "lambertw(" + args + ")"
 
 
+    def _nested_binary_math_func(self, expr):
+        args = self._print(expr.args[0])
+        args += ', %s' % self._print(expr.func(*expr.args[1:]))
+
+        return '{name}({args})'.format(
+            name=self.known_functions[expr.__class__.__name__],
+            args=args
+        )
+
+    def _print_Max(self, expr):
+        return self._nested_binary_math_func(expr)
+
+    def _print_Min(self, expr):
+        return self._nested_binary_math_func(expr)
```

Comment:
You can do
```python
_print_Max = _print_Min =\
    _nested_binary_math_func
```
as they just pass the call to `_nested_binary_math_func`.

Reply:
Done @ylemkimon , @bjodah .